src/pages/Dashboard/components/RecentTodos/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { Todo } from '@/types/interface';
+import { ArrowRightIcon, TodoRecentlyIcon } from '@assets';
+import TodoList from '@components/TodoList';
+import { useState } from 'react';
+import { mockTodosData } from '../mockData';
+
+function RecentTodos() {
+  const todosData = mockTodosData
+    .sort((a, b) => {
+      const dateA = new Date(a.createdAt).getTime();
+      const dateB = new Date(b.createdAt).getTime();
+      return dateA - dateB;
+    })
```

Comment:
```suggestion
.sort((a, b) => new Date(a.createdAt) - new Date(b.createdAt))
```
[Suggestion] 이렇게도 쓸 수 있을 것 같아요~